azure-pipelines.yml

```diff
@@ -78,10 +78,18 @@ jobs:
           MB_PYTHON_VERSION: "3.8"
           NP_BUILD_DEP: "numpy==1.17.3"
           NIGHTLY_BUILD: "true"
+        py_3.8_universal2:
+          MB_PYTHON_VERSION: "3.8"
+          NP_BUILD_DEP: "numpy==1.17.3"
+          NIGHTLY_BUILD: "true"
```

Comment:
should this include `PLAT: universal2` like the py_3.10_universal2 config

Reply:
I can't believe I missed this. Sending in a new PR.